.github/workflows/build.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         uses: actions/setup-java@v3
         with:
           java-version: ${{ matrix.java }}
-          distribution: 'temurin'
+          distribution: 'zulu'
```

Comment:
Using zulu isn't recommended for CI due to certain tweaks being worse than temurin, Also consider including `check-latest: true` below distribution.

Reply:
Never heard of that about `zulu`. Do you have any links for me, where I can read about it, because I'm interested about the background of the named tweaks?

I suggested `zulu` because I'm using it myself for a long time now and it is used by many bigger projects for CI like `netty`, `lombok` and `guava`...

Reply:
zulu was previously chosen due to faster update times. In general I think we dont neccessarily need the very latest version of the jdk, testing the build and tests qith multiple jdks is just there to ensure basic compatibility with the lts versions.